src/main/resources/bootstrap.yaml

```diff
@@ -17,3 +17,5 @@ spring:
         flyway-password: flyway.password
         blob-router-POSTGRES-PASS: DB_PASSWORD
         storage-reform-connection-string: storage.reform.connection-string
+        reports-email-username: SMTP_USERNAME
+        reports-email-password: SMTP_PASSWORD
```

Comment:
if those are for smtp auth, then we are missing enlisting them in `values.yaml`. on the other hand, since we are bootstrapping them directly to spring boot properties - do we even need extra env var defined?